README.md

```diff
@@ -28,9 +28,9 @@ To compile this project, use the dotnet CLI (https://github.com/dotnet/cli) to e
 
 `dotnet restore`
 
-`dotnet publish .\\src\\Docker.PowerShell -o .\\src\\Docker.PowerShell\\bin\\Module\\Docker.PowerShell -r win`
+`dotnet publish .\\src\\Docker.PowerShell -o .\\src\\Docker.PowerShell\\bin\\Module\\Docker -r win`
 
-This will produce the PowerShell module  at ".\src\Docker.PowerShell\bin\Module\Docker.PowerShell" in the project folder.  If Visual Studio is not already installed, the dotnet CLI may require an additional installation of the .NET 4.6 developer pack (https://www.microsoft.com/en-us/download/details.aspx?id=48136).
```

Comment:
The assembly will still be called Docker.PowerShell.dll, so the path should stay the same. It's just the PowerShell module that should be called "Docker". Does that make sense?